src/main/java/net/atos/zac/app/taken/converter/RESTTaakHistorieConverter.java

```diff
@@ -69,8 +71,18 @@ private RESTTaakHistorieRegel convert(final HistoricTaskLogEntry historicTaskLog
 
     private RESTTaakHistorieRegel convertData(final HistoricTaskLogEntryType type, final String data) {
         return switch (type) {
-            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(CREATED_ATTRIBUUT_LABEL);
-            case USER_TASK_COMPLETED -> new RESTTaakHistorieRegel(COMPLETED_ATTRIBUUT_LABEL);
+            case USER_TASK_CREATED -> new RESTTaakHistorieRegel(
+                    CREATED_ATTRIBUUT_LABEL,
+                    null,
+                    CREATED_ATTRIBUUT_LABEL,
+                    null
+            );
+            case USER_TASK_COMPLETED -> new RESTTaakHistorieRegel(
+                    COMPLETED_ATTRIBUUT_LABEL,
+                    CREATED_ATTRIBUUT_LABEL,
```

Comment:
Can we have other states?

Reply:
That's a question for you to find the answer I'm afraid. ;-) Anyway, I would focus on the states mentioned in the user story.

Reply:
States will be determined by `HistoricTaskLogEntryType`.

Reply:
Currently we use open/closed only. There are other states in Flowable, but we don't use them